src/Analyzer.PowerShellRuleEngine/Analyzer.PowerShellRuleEngine.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Add enforcement of the xml documentation to this project. See the Build properties of the other projects in the solution.